cmd/tailscale/cli/serve.go

```diff
@@ -32,7 +32,16 @@ import (
 	"tailscale.com/version"
 )
 
-var serveCmd = newServeCommand(&serveEnv{lc: &localClient})
+var serveCmd = func() *ffcli.Command {
+	se := &serveEnv{lc: &localClient}
+	// This flag is used to switch to an in-development
+	// implementation of the tailscale funnel command.
+	// See https://github.com/tailscale/tailscale/issues/7844
+	if os.Getenv("TAILSCALE_FUNNEL_DEV") == "on" {
```

Comment:
consider using `envknob.UseWIPCode` that you found ?

Reply:
👍 updated in f128e97331367222543dd9f26c1784f1dd43bda2